sql/updates/0.11.0--0.12.0.sql

```diff
@@ -66,3 +66,14 @@ CREATE TABLE IF NOT EXISTS _timescaledb_catalog.installation_metadata (
 SELECT pg_catalog.pg_extension_config_dump('_timescaledb_catalog.installation_metadata', $$WHERE key='exported_uuid'$$);
 
 INSERT INTO _timescaledb_catalog.installation_metadata SELECT 'install_timestamp', to_timestamp(0);
+
+-- We have to create the force restart function here, in the update script for the first version that should include a background worker  
```

Comment:
the comment here could be improved as above. I would also be explicit that we are explaining why this is not a plain start but rather a start_or_restart.